keepalived-vip/utils.go

```diff
@@ -59,6 +60,16 @@ func getNodeInfo(nodes []string) (*nodeInfo, error) {
 	}, nil
 }
 
+// getEffectiveIP returns the forcedIP if it was given on the cmdline.
+// If forcedIP is not set it will return the node address
+func getEffectiveIP(nodes []string, forcedIP string) (string, error) {
```

Comment:
can you clarify what you're trying...? (Is not possible to assign just one IP (this can be running as DaemonSet)